docs/api/qiskit-ibm-runtime/qiskit_ibm_runtime.Estimator.md

```diff
@@ -73,13 +70,9 @@ Initializes the Estimator primitive.
 
 ## Attributes
 
-<span id="estimator-circuits" />
-
-### circuits
-
 <span id="qiskit_ibm_runtime.Estimator.circuits" />
 
-`tuple[qiskit.circuit.quantumcircuit.QuantumCircuit, ...]`
```

Comment:
This is a good change - we were saying the return type twice for some reason.

Before:

<img width="686" alt="Screenshot 2023-10-20 at 5 22 15 PM" src="https://github.com/Qiskit/documentation/assets/14852634/27d4e61b-dcec-467f-8ac4-8fab1032c934">

This PR:
<img width="478" alt="Screenshot 2023-10-20 at 5 22 35 PM" src="https://github.com/Qiskit/documentation/assets/14852634/4eb2b3c1-6604-414c-b37a-a7eedf727165">